fv3core/stencils/ytp_v.py

```diff
@@ -56,7 +57,7 @@ def _compute_stencil(
     dya: FloatFieldIJ,
     rdy: FloatFieldIJ,
 ):
-    from __externals__ import i_end, i_start, j_end, j_start, jord, namelist
+    from __externals__ import i_end, i_start, j_end, j_start, jord
```

Comment:
i am still very bugged about having end before start. Why do we keep leaving this in?

Reply:
it keeps sneaking back in with refactors and merges, good eye, I can update

Reply:
the linter won't let me! :)